### ISSUE TITLE
JAWS screen reader does not read associated dropdown label on focus .It  reads label after dropdown is expanded post focus click on dropdown.

### ISSUE THREAD

**Issue by v-ajrai** [microsoft/fluentui#16510]
JAWS screen reader does not read associated dropdown label on focus .It  reads label after dropdown is expanded post focus click on dropdown.

### Environment Information

Tool: JAWS
Browser: Chrome Version 87.0.4280.88 (Official Build) (64-bit)

### Describe the issue:

JAWS screen reader does not read associated dropdown label on focus .It  reads label after dropdown is expanded post focus click on dropdown.

### Please provide a reproduction of the issue in a codepen:
https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown
![dropdown_JAWS_ISSUE](https://user-images.githubusercontent.com/62233982/104888347-9f70d000-5992-11eb-837b-26e542650e4e.PNG)

#### Actual behavior:

JAWS screen reader does not read associated dropdown label on focus .It  reads label after dropdown is expanded post focus click on dropdown.

#### Expected behavior:

JAWS screen reader should read associated dropdown label on focus prior to dropdown expansion.

**Comment by v-ajrai** [comment]
Any ETA on this ?

**Comment by v-ajrai** [comment]
Could you please let us know when we can get fix for this.?

**Comment by gouttierre** [comment]
@v-ajrai - Thank you for the feedback and issue submission. This seems to be a duplicate request to #16490. To set expectation the engineering team will review it as soon as bandwidth allows and update the issue. Thanks!

cc @khmakoto, @smhigley

**Comment by v-ajrai** [comment]
@gouttierre Could you please let us know when we could get fix for this?
